web/src/app/users/UserCalendarSubscriptionList.tsx

```diff
@@ -56,18 +62,20 @@ export default function UserCalendarSubscriptionList(props) {
   if (!_.get(data, 'user.id')) return loading ? <Spinner /> : <ObjectNotFound />
 
   // sort by schedule names, then subscription names
-  const subs = data.user.calendarSubscriptions.slice().sort((a, b) => {
-    if (a.schedule.name < b.schedule.name) return -1
-    if (a.schedule.name > b.schedule.name) return 1
+  const subs: UserCalendarSubscription[] = data.user.calendarSubscriptions
+    .slice()
+    .sort((a: UserCalendarSubscription, b: UserCalendarSubscription) => {
+      if ((a?.schedule?.name ?? '') < (b?.schedule?.name ?? '')) return -1
+      if ((a?.schedule?.name ?? '') > (b?.schedule?.name ?? '')) return 1
 
-    if (a.name > b.name) return 1
-    if (a.name < b.name) return -1
-  })
+      if (a.name > b.name) return 1
+      if (a.name < b.name) return -1
+    })
 
-  const subheaderDict = {}
-  const items = []
+  const subheaderDict: { [key: string]: boolean } = {}
+  const items: FlatListListItem[] = []
 
-  function renderOtherActions(id) {
+  function renderOtherActions(id: string): ReactElement {
```

Comment:
```suggestion
  function renderOtherActions(id: string): JSX.Element {
```

Essentially the same thing but we use JSX.Element more across the app